cyences_app_for_splunk/default/macros.conf

```diff
@@ -960,6 +986,9 @@ iseval = 0
 definition = search *
 iseval = 0
 
+[cs_authentication_success_login_outside_home_country_filter]
```

Comment:
Rename to `cs_successful_vpn_login_outside_home_country_filter`

Reply:
Changed name to "cs_authentication_successful_vpn_login_outside_home_country_filter"